<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputWindow</class>
 <widget class="QWidget" name="InputWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>10</y>
     <width>311</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="NameInput"/>
    </item>
    <item>
     <widget class="QLineEdit" name="PriceInput"/>
    </item>
    <item>
     <widget class="QLineEdit" name="ShopInput"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="NameLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>32</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Название товара:</string>
   </property>
  </widget>
  <widget class="QLabel" name="PriceLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>88</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Цена товара:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ShopLabel">
   <property name="geometry">
    <rect>
     <x>78</x>
     <y>141</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Магазин:</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>210</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="EnterButton">
      <property name="text">
       <string>Ввести</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ClearButton">
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="ItemsRemain">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>186</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Осталось ввести: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
